source/_common.R

```diff
@@ -81,7 +91,7 @@ get_var = function(name) {
 
 # See: https://stackoverflow.com/questions/76732162/is-there-a-way-to-detect-the-chunk-language-when-setting-knitr-chunk-options
 set_true_for_matching_lang <- function(options, field) {
-    if (is.na(options[[field]])) {
+    if (!is.logical(options[[field]])) {
```

Comment:
This change seems fine, although the current version feels more precise (replace default value with X).

This could be a nice change if you also set the default value to NULL instead of NA.